frontend/.env.local.example

```
@@ -18,6 +18,8 @@ FACEBOOK_CLIENT_SECRET=your_facebook_app_secret_here
 
 # Resend Email API
 RESEND_API_KEY=re_your_resend_api_key_here
+# Sender for transactional emails. Must exactly match a verified Resend domain.
+EMAIL_FROM=Disscount <noreply@disscount.me>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the RESEND_API_KEY key

(UnorderedKey)